homeassistant/components/atag/climate.py

```diff
@@ -0,0 +1,107 @@
+"""Initialization of ATAG One climate platform."""
+from typing import List, Optional
+
+from homeassistant.components.climate import ClimateDevice
+from homeassistant.components.climate.const import (
+    CURRENT_HVAC_HEAT,
+    CURRENT_HVAC_IDLE,
+    DEFAULT_MAX_TEMP,
+    DEFAULT_MIN_TEMP,
+    HVAC_MODE_AUTO,
+    HVAC_MODE_HEAT,
+    HVAC_MODE_OFF,
+    SUPPORT_TARGET_TEMPERATURE,
+)
+from homeassistant.const import ATTR_TEMPERATURE, TEMP_CELSIUS
+from homeassistant.helpers.restore_state import RestoreEntity
+
+from . import DOMAIN, AtagEntity
+
+
+async def async_setup_platform(hass, _config, async_add_devices, _discovery_info=None):
+    """Atag updated to use config entry."""
+    pass
+
+
+async def async_setup_entry(hass, entry, async_add_devices):
+    """Load a config entry."""
+    atag = hass.data[DOMAIN][entry.entry_id]
+    async_add_devices([AtagThermostat(atag, "Thermostat")])
+
+
+# pylint: disable=abstract-method
```

Comment:
Do we need to disable this pylint error?

Reply:
not anymore it seems 👍